### PR TITLE
created landing page and altered button component

### PR DESCRIPTION
**Pull request by klevun7**
Screenshot 2024-01-21 at 10 29 14 PM

- created landing page and altered Button.jsx to handle props of `buttonText`
- I need some help with positioning the image and the elements without the use of setting margins and padding I think it has something to do with the width and height I set for the entire component.

### CODE REVIEW

File: src/components/home/Landing.jsx [hunk 1]
```diff
@@ -1,5 +1,53 @@
...
+          <Link
+            href="https://www.instagram.com/ucrprishi/?hl=en"
+            target="_blank"
+          >
+            <Button buttonText={"Join Us"} />
+          </Link>
+        </div>
+      </div>
+
+      <div className="flex relative w-screen -top-24 -right-24">
```
Comment (pondohoo) [inline src/components/home/Landing.jsx:40]: justify-center items-center so the image shrinks towards the middle when you shrink the screen
(adjust -top-24 -right-24 to look the same as it does now)

File: src/components/home/Landing.jsx [hunk 2]
```diff
@@ -1,5 +1,53 @@
...
+import logo from "@/public/prishi-logo.webp";
+import landing from "@/public/landing.webp";
+import Button from "@/components/Button";
+import Link from "next/link";
+
 const Landing = () => {
-  return <div>Landing</div>;
+  return (
+    <div className="flex h-screen w-screen ">
+      <div className="flex flex-col w-7/12 justify-center gap-3 pl-12">
```
Comment (pondohoo) [inline src/components/home/Landing.jsx:10]: u can honestly make it w-8/12 so the largest text doesn't wrap until smaller screen sizes

**Comment by klevun7** [comment]
Screenshot 2024-01-22 at 11 45 39 PM
- implemented changes to landing image to shrink towards the center of the screen.
- adjusted width of text to not wrap on smaller screens.
- adjusted padding of outline around image to fit closer to figma spec.
- still have problem of image moving as left margin of text div is changed.

File: src/components/home/Landing.jsx [hunk 3]
```diff
@@ -1,5 +1,53 @@
...
 const Landing = () => {
-  return <div>Landing</div>;
+  return (
+    <div className="flex h-screen w-screen">
+      <div className="flex flex-col w-8/12 justify-center gap-3">
+        <div className="flex justify-center">
+          <Image src={logo} alt="rishi-logo" className="w-24 h-auto" />
+        </div>
+
+        <div className="flex justify-center">
```
Comment (pondohoo) [inline src/components/home/Landing.jsx:15]: you don't need to wrap the "Project RISHI at UCR" container in this extra flex container. delete this extra flex container and just add text-center to the Project RISHI at UCR container

File: src/components/home/Landing.jsx [hunk 4]
```diff
@@ -1,5 +1,53 @@
+import Image from "next/image";
+import logo from "@/public/prishi-logo.webp";
+import landing from "@/public/landing.webp";
+import Button from "@/components/Button";
+import Link from "next/link";
+
 const Landing = () => {
-  return <div>Landing</div>;
+  return (
+    <div className="flex h-screen w-screen">
```
Comment (pondohoo) [inline src/components/home/Landing.jsx:9]: add ml-6 here to prevent it from hitting the border of the screen

**Comment by klevun7** [comment]
Screenshot 2024-01-25 at 1 19 21 PM

- removed extra flex-container around text.
- added ml-6 to prevent hitting border of screen.

**Review by pondohoo — APPROVED** [review]

**Pull request merged**